uk.co.compucorp.civicrm.hrleaveandabsences/hrleaveandabsences.php

```diff
@@ -104,6 +104,8 @@ function hrleaveandabsences_civicrm_permission(&$permissions) {
   $prefix = ts('CiviHRLeaveAndAbsences') . ': '; // name of extension or module
   $permissions['access leave and absences'] = $prefix . ts('Access Leave and Absences');
   $permissions['administer leave and absences'] = $prefix . ts('Administer Leave and Absences');
+  $permissions['access leave and absences in ssp'] = $prefix . ts('Access CiviHR Leave and Absences');
+  $permissions['manage leave and absences in ssp'] = $prefix . ts('Manage CiviHR Leave and Absences');
```

Comment:
If you check the permissions page on Drupal, you'll see that, before this change, the permissions were grouped into 2 different sections. The first 2 were grouped under CiviCRM and the last 2 were groups under civiHR employee portal. This helped the user identify which ones were applied to the SSP and which ones to the Admin. Now there's basically no difference.

I suggested you rename the SSP permissions to:
- Access Leave and Absences in SSP
- Manage Leave and Absences in SSP